modules/mvt/docs/api-reference/mapbox-vector-tile-loader.md

```diff
@@ -83,10 +83,12 @@ const geoJSONfeatures = await load(url, MVTLoader);
-| --------------- | ------------------------------------------ | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| mvt.coordinates | String                                     | `local`      | When set to `wgs84`, parser will return a flat array of GeoJSON representations of the features with coordinates decoded from provided tile index. When set to `local`, parser will return a flat array of GeoJSON objects with local coordinates decoded from tile origin. |
-| mvt.tileIndex   | Object ({x: number, y: number, z: number}) | _No default_ | Mandatory with `wgs84` coordinates option. An object containing tile index values (x, y, z) to decode WGS84 coordinates.                                                                                                                                                    |
+| Option            | Type                                       | Default      | Description                                                                                                                                                                                                                                                                 |
+| ----------------- | ------------------------------------------ | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| mvt.coordinates   | String                                     | `local`      | When set to `wgs84`, parser will return a flat array of GeoJSON representations of the features with coordinates decoded from provided tile index. When set to `local`, parser will return a flat array of GeoJSON objects with local coordinates decoded from tile origin. |
+| mvt.layerProperty | String                                     | `layerName`  | The layer name is added to `feature.properties[layerProperty]` (first adds a `feature.property` object is created if it did not already exist). If set to `null`, it won't add layer name to any property.                                                                  |
+| mvt.layers        | String[]                                   | _No default_ | Optional list of layer names. If supplied, only features belonging to the named layers will be included in the output. If not supplied, features from all layers are returned.                                                                                              |
```

Comment:
Nit: I think it makes sense to specify a default so that the app can contain code that conditionally sets this prop without if statements.  Perhaps `null` or `false`?